File: Foundation/Basics/geetingstarted/8_reverse.cpp
#include <iostream>
using namespace std;

void RevNum(int n){
   // int ctr=0;
    int num = n;
    while(num){
        int res = num%10;
        cout<<res<<endl;
        num /= 10;
        }
}

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
    RevNum(n);
    return 0;
}

File: Foundation/Basics/geetingstarted/8_reverse.java
import java.util.*;
   
   public class Main{
       
   public static Scanner scn = new Scanner(System.in);
   
   public static void RevNum(int n){
       int num = n;
       while(num>0){
           int res = num%10;
           System.out.println(res);
           num /= 10;
       }
   }
   
   public static void main(String[] args) {
     // write your code here  
     int n = scn.nextInt();
     RevNum(n);
    }
   }